### ISSUE TITLE
ImageScan not working with current version

### ISSUE THREAD

**Issue by kirschkern** [mebjas/html5-qrcode#722]
I have an image than can be scanned at https://blog.minhazav.dev/research/html5-qrcode
However when using https://scanapp.org/ or including the 2.3.7 build into my package.json, the qr code at this particular image will not be recognized.
I've now included the script from the first demo into my project which works fine.

**Comment by iaheck** [comment]
Hello, I'm having the same issue with another image. Maybe the demo page does some of this scaling under the hood. But in my tests it works when I use a square image up to 430 px. @mebjas, are we missing some extra step?
I noticed that there is a downsample if the image is bigger than the container, but it seems that that's only for displaying it and not for reading the QR code

**Comment by iaheck** [comment]
@kirschkern , did you find any workaround? thank you both guys

**Comment by kirschkern** [comment]
No, I have no other solution except using the old version from https://blog.minhazav.dev/research/html5-qrcode

**Comment by iaheck** [comment]
hey @kirschkern , after some deep debugging I found out that the website was using [Barcode Detection API](https://developer.mozilla.org/en-US/docs/Web/API/Barcode_Detection_API) and my app was not. Barcode requires HTTPS and I was trying on localhost without any certificate so it couldn't use it. I used with ngrok to access to my site in a secure context and it worked!

Barcode does not work with firefox yet, so your image is not decoded in https://scanapp.org/ if you use that browser. So it's the same issue and when you publish your app with https it will work on supported browsers (oddly, it says that Safari it's not supported yet but it is)
